docs/spec/spec.md

```diff
@@ -254,12 +254,6 @@ spec:
   # Name of the service account the code should run as.
   serviceAccountName: ...
 
-  # The Revision's level of readiness for receiving traffic.
-  # This may not be specified at creation (defaults to Active),
-  # and is used by the controllers and activator to enable
-  # scaling to/from 0.
-  servingState: Active | Reserve | Retired
```

Comment:
Can you leave this in, but note it's deprecation and the fact that it is not unused?